onnxruntime/core/providers/cpu/tensor/upsample.h

```diff
@@ -79,7 +79,7 @@ class UpsampleBase {
       const Tensor* scale;
       bool get_scale = info.TryGetConstantInput(scales_input_idx_, &scale);
 
-      if (get_scale) {
+      if (get_scale&& scale->Shape().Size() > 0) {
```

Comment:
Wouldn't have thought this possible. If there's a initializer how would it have a size of 0?

Reply:
scales is added as an empty tensor of size zero... some pytorch test models were failing because of this missing check...